docs/stdlib/json.rst

```diff
@@ -245,9 +240,10 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
 .. eql:operator:: jsonobjdest: json [ str ] -> json
 
-    JSON object property access.
+    Accesses the value of a :eql:type:`json` from its :eql:type:`str` key:
 
-    The fields of JSON *objects* can also be accessed via ``[]``:
+    This results in a :eql:type:`json` type. The fields of any JSON object may
+    also be accessed via ``[]``:
```

Comment:
@vpetrovykh Can you help me understand the intent of the second sentence here:

> The fields of any JSON object may also be accessed via ``[]``

The "also" suggests we are presenting a new idea, but this sounds like we're just describing the operator again. I think I'm probably misunderstanding.